src/context.rs

```diff
@@ -421,16 +393,9 @@ pub unsafe fn dc_get_info(context: &Context) -> *mut libc::c_char {
         real_msgs,
         deaddrop_msgs,
         contacts,
-        context
-            .get_dbfile()
-            .as_ref()
-            .map_or(unset, |p| p.to_str().unwrap()),
+        context.get_dbfile().to_str().unwrap(),
```

Comment:
`.get_dbfile().display()` is safer isn't it?  I think we should not have an unwrap here